html/status.html

```diff
@@ -597,8 +597,10 @@
 				}
 
 				var decerr = state['ad_decerr'][i];
+				var decerrpre = oldstate['ad_decerr'][i];
+				decerrpre = decerr - decerrpre;
 				if (decerr > 0)
-					myTable += "DEC-ERRORS: " + decerr + "<br />";
+					myTable += "DEC-ERRORS: " + decerr + " (" + decerrpre + ")<br />";
```

Comment:
Should this logic be used for CC/BER counters too?

Reply:
It could be done...

Reply:
Sure